config/environments/development.rb

```diff
@@ -60,12 +60,10 @@
 
   config.aws_region = ENV['AWS_REGION']
 
-  config.hub_environments_legacy = { 'development': 'development-bucket' }
-
   config.hub_environments = {
     'development': {
       'bucket': 'development-bucket',
-      'hub-config-host': 'http://config-service.dev',
+      'hub_config_host': 'http://localhost:50240',
       'secure-header': 'false'
```

Comment:
here too

Reply:
@jkunle sorry, I resolved the other conversations but I didn't realise my push had failed :( is up there now!